abci/client/socket_client.go

```diff
@@ -92,6 +94,14 @@ func (cli *socketClient) OnStop() {
 	cli.flushQueue()
 }
 
+// RunForever waits for an interrupt signal and stops the node.
+func (cli *socketClient) RunForever() {
```

Comment:
Lets rename from `RunForever` to catch interrupt?

Reply:
sure:smiley: